lib/proofer/vendor/mock_response.rb

```diff
@@ -0,0 +1,8 @@
+require 'hashie'
+
+module Proofer
+  module Vendor
+    class MockResponse < Hashie::Mash
```

Comment:
instead of an entire dependence on Hashie, what about just using an OpenStruct here?